demo/src/app/search-filters-consumer.component.ts

```diff
@@ -8,14 +8,58 @@ import { Jurisdiction, CaseType } from '@hmcts/ccd-case-ui-toolkit';
 })
 export class SearchFiltersConsumerComponent implements OnInit {
```

Comment:
We do not need this one in demo app if we have a wrapper. Please remove.

Reply:
Done.